src/sentry/static/sentry/app/utils.tsx

```diff
@@ -33,8 +33,8 @@ export function valueIsEqual(value, other, deep) {
   return false;
 }
 
-function objectMatchesSubset(obj, other, deep) {
-  let k;
+function objectMatchesSubset(obj?: object, other?: object, deep?: boolean): boolean {
+  let k: string | number;
```

Comment:
Why number, shouldn't it always be string?

Reply:
@lynnagara I thought so too. I used vscode to infer the type for me:

<img width="1680" alt="Screen Shot 2019-07-29 at 11 54 28 PM" src="https://user-images.githubusercontent.com/139499/62100176-a4056e00-b25e-11e9-9278-feedd1375280.png">

Reply:
Is this because array is a valid type of object (keys are just numbers instead of strings?). In our usage of this function I think it should only be an object with key as string.

Reply:
I made `k` a string.